CSharp/DevVmPowershell/Helpers/Implementations/InstanceSettingsHelper.cs

```diff
@@ -4,35 +4,60 @@
 using Relativity.Services.ServiceProxy;
 using System;
 using System.Linq;
+using System.Net.Http;
+using Helpers.RequestModels;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Helpers.Implementations
 {
 	public class InstanceSettingsHelper : IInstanceSettingsHelper
 	{
 		private ServiceFactory ServiceFactory { get; }
+		private string InstanceAddress { get; set; }
+		private string AdminUsername { get; set; }
+		private string AdminPassword { get; set; }
 
-		public InstanceSettingsHelper(IConnectionHelper connectionHelper)
+		public InstanceSettingsHelper(IConnectionHelper connectionHelper, string instanceAddress, string adminUsername, string adminPassword)
 		{
 			ServiceFactory = connectionHelper.GetServiceFactory();
+			InstanceAddress = instanceAddress;
+			AdminUsername = adminUsername;
+			AdminPassword = adminPassword;
 		}
 
 		public int CreateInstanceSetting(string name, string section, string description, string value)
 		{
 			try
 			{
-				using (IInstanceSettingManager instanceSettingManager = ServiceFactory.CreateProxy<IInstanceSettingManager>())
+				HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+				InstanceSettingManagerCreateRequest instanceSettingManagerUpdateRequest = new InstanceSettingManagerCreateRequest
 				{
-					InstanceSetting newInstanceSetting = new InstanceSetting
+					instanceSetting = new RequestModels.instanceSetting
 					{
-						Section = section,
 						Name = name,
-						Description = description,
+						Section = section,
+						Machine = "",
+						ValueType = "Text",
 						Value = value,
-					};
-					int instanceSettingArtifactId = instanceSettingManager.CreateSingleAsync(newInstanceSetting).Result;
-					Console.WriteLine("Successfully Created Instance Setting");
-					return instanceSettingArtifactId;
+						InitialValue = value,
+						Encrypted = false,
+						Description = description,
+						Keywords = "",
+						Notes = ""
+					}
+				};
+				string createRequest = JsonConvert.SerializeObject(instanceSettingManagerUpdateRequest);
+				HttpResponseMessage createResponse = RestHelper.MakePost(httpClient, Constants.Connection.RestUrlEndpoints.InstanceSettings.endpointUrl, createRequest);
```

Comment:
We recently made change to the RestHelper to have an interface. why are we not using that?

- [ ] Use IRestHelper interface